ros2cli/ros2cli/daemon/__init__.py

```diff
@@ -92,6 +96,65 @@ def shutdown_handler():
             server.server_close()
 
 
+class _DirectNode:
```

Comment:
@ivanpauno `DaemonNode` ?

Reply:
Can be confused with https://github.com/ros2/ros2cli/blob/3feb662510fc2b84c9f6aed8ee7af2f6aa82b8f8/ros2cli/ros2cli/node/daemon.py#L96.

I have some options:
- AutoResettingNode
- NetworkAutoResettingNode
- NetworkAwareNode
- DaemonDirectNode
- NodeWithAutoresettingPower